releasenotes/notes/no-sympy-19687d5ca814042c.yaml

```diff
@@ -0,0 +1,10 @@
+---
+upgrade:
+  - |
+    ``sympy`` is no longer a requirement for installing Qiskit. After the migration to
+    a Rust based symbolic engine for the :class:`.ParameterExpression` class the uses
+    of Sympy are isolated to some visualization utilities and the
```

Comment:
```suggestion
    a Rust based symbolic engine for the :class:`.ParameterExpression` class, the uses
    of sympy are isolated to some visualization utilities and the
```

Reply:
I think we're both wrong, the correct capitalization in the docs is apparently "SymPy": https://docs.sympy.org/latest/index.html

Reply:
Updated in: https://github.com/Qiskit/qiskit/pull/14396/commits/fc83c8f954313c62cbceb009978dda9363e38ff8